LoopKitUI/Extensions/UIColor+LoopKit.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  UIColor.swift
+//  LoopKitUI
+//
+//  Created by Anna Quinlan on 7/1/20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+import UIKit
+
+extension UIColor {
+    @nonobjc static let COBTintColor: UIColor = {
+        if #available(iOSApplicationExtension 13.0, iOS 13.0, *) {
+            return UIColor(dynamicProvider: { (traitCollection) in
+                // If we're in accessibility mode, return the system color
+                guard case .normal = traitCollection.accessibilityContrast else {
+                    return .systemGreen
+                }
+
+                switch traitCollection.userInterfaceStyle {
+                case .unspecified, .light:
+                    return UIColor(red: 99 / 255, green: 218 / 255, blue: 56 / 255, alpha: 1)
```

Comment:
Did these colors come from the Figma color palette?

Reply:
This was (and still is) in Loop and was copy-pasted by me, so no new colors added here. Would it make sense to change it to be aligned with the Figma? I'd be concerned about changing it because it's a color scheme that DIY uses as well

Reply:
ok, in that case it shouldn't block this PR, but I would make sure that the designers approve the colors (per our conversation about colors today).

Reply:
I think any work involving color changes will be part of whatever mechanism we design to support configurable color palettes between Tidepool and DIY.